app/assets/stylesheets/common/base/topic-post.scss

```
@@ -1349,6 +1349,8 @@ a.mention-group {
   display: flex;
   align-items: center;
   margin-right: 2em;
+  max-width:400px;
+  overflow:hidden;
```

```suggestion
  max-width: 400px;
  overflow: hidden;
```

I think you need the spaces here for linting to pass.

---

Didn't mean to resolve, will fix ASAP.